### PR TITLE
Add FETT263_SPECIAL_ABILITIES

### PR DESCRIPTION
**Pull request by fdarosa2663**

### CODE REVIEW

File: props/saber_fett263_buttons.h [hunk 1]
```diff
@@ -5653,15 +5724,45 @@ SaberFett263Buttons() : PropBase() {}
           current_menu_angle_ = fusor.angle2();
           return true;
         }
-        if (menu_) MenuDialIncrement(1);
+        if (menu_) {
+	  MenuDialIncrement(1);
+#ifdef FETT263_SPECIAL_ABILITIES
+#if NUM_BUTTONS == 1
+        } else {
```
Comment (profezzorn) [inline props/saber_fett263_buttons.h:5731]: I think the "} else {" should be befor the #if since it is both in the if and else branches.
(same below)

Comment (fdarosa2663) [reply props/saber_fett263_buttons.h:5730]: Updated

File: props/saber_fett263_buttons.h [hunk 2]
```diff
@@ -5680,6 +5780,20 @@ SaberFett263Buttons() : PropBase() {}
             MenuDialIncrement(1);
             return true;
           }
+#ifdef FETT263_SPECIAL_ABILITIES
+#if NUM_BUTTONS == 1
+        } else {
```
Comment (profezzorn) [inline props/saber_fett263_buttons.h:5785]: move out

File: props/saber_fett263_buttons.h [hunk 3]
```diff
@@ -5653,15 +5724,44 @@ SaberFett263Buttons() : PropBase() {}
...
         if (wav_player && wav_player->isPlaying()) {
           current_menu_angle_ = fusor.angle2();
           return true;
         }        
-        if (menu_) MenuDialIncrement(-1);
+        if (menu_) {
+	  MenuDialIncrement(-1);
+#ifdef FETT263_SPECIAL_ABILITIES
+#if NUM_BUTTONS == 1
+        } else {
```
Comment (profezzorn) [inline props/saber_fett263_buttons.h:5753]: move out

File: props/saber_fett263_buttons.h [hunk 4]
```diff
@@ -5699,7 +5813,21 @@ SaberFett263Buttons() : PropBase() {}
             MenuDialIncrement(-1);
             return true;
           }
-        }
+#ifdef FETT263_SPECIAL_ABILITIES
+#if NUM_BUTTONS == 1
+        } else {
```
Comment (profezzorn) [inline props/saber_fett263_buttons.h:5818]: move out

File: props/saber_fett263_buttons.h (hunk 1 above)
Comment (fdarosa2663) [reply props/saber_fett263_buttons.h]: updated

File: props/saber_fett263_buttons.h (hunk 2 above)
Comment (fdarosa2663) [reply props/saber_fett263_buttons.h]: updated

File: props/saber_fett263_buttons.h (hunk 3 above)
Comment (fdarosa2663) [reply props/saber_fett263_buttons.h]: updated

File: props/saber_fett263_buttons.h (hunk 4 above)
Comment (fdarosa2663) [reply props/saber_fett263_buttons.h]: updated

**Pull request merged**